src/platform/i386/runscripts/micro_boot.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cp micro_boot.o llboot.o
+./cos_linker "llboot.o, :" ./gen_client_stub
```

Comment:
You are not suppose to change the microbenchmarks. You can have them modified on your local environment but not push it to the main repo.

Reply:
We can talk about if the FPU tests should be part of microbenchmarks (as they are now), or should be a separate test. As some platforms won't have fpu, and others will, but we won't enable it, it perhaps makes sense for it to be a separate test?

Reply:
Sorry, I did not update the branch before the pull request. I did not make any modifications to the file here and below. I will fix them.